playwright/e2e/atk_entity/atk_user.spec.js

```diff
@@ -36,18 +36,18 @@ test.describe('User tests.', () => {
   test('(ATK-PW-1100) Create and delete user with Drush. @ATK-PW-1100 @user @drush @smoke @alters-db', async ({ page }) => { // eslint-disable-line no-unused-vars
     const testId = 'ATK-PW-1100'; // eslint-disable-line no-unused-vars
 
-    await atkCommands.deleteUserWithEmail(userEtherealAccount.userEmail, ['--delete-content']);
+    await atkCommands.deleteUserByEmail(userEtherealAccount.userEmail, ['--delete-content']);
```

Comment:
We can't change the API because it also is used in Automated Testing Kit (the Drupal 10 version).